test_scripts/API/VehicleData/Refactor_Fuel_Information_Related_Vehicle_Data/012_GetVehicleData_with_deprecated_fuelLevel_and_fuelLevel_State.lua

```diff
@@ -0,0 +1,28 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal:https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0256-Refactor-Fuel-Information-Related-Vehicle-Data.md
+-- Description: SDL successfully process GetVehicleData with deprecated fuelLevel and fuelLevel_State
```

Comment:
But GetVehicleData has already checked in 010 and 011, so propose to remove this script.
In my opinion this is overhead.